core/src/lib.rs

```diff
@@ -66,6 +66,7 @@ pub mod upgrade;
 
 pub use connection::{Connected, ConnectedPoint, Endpoint};
 pub use identity::PublicKey;
+pub use instant::{Duration, Instant};
```

Comment:
If I am not mistaken this is the first non-libp2p-related re-export in `libp2p-core`, correct?

If so, I wonder whether it is actually needed. Neither of these two types are exposed on the public API of `libp2p-core`, right? Do versions actually have to match across libp2p crates?